### ISSUE TITLE
rename_cluster_node command is broken.

### ISSUE THREAD

**Issue by lukebakken** [rabbitmq/rabbitmq-cli#457]
Reported in this `rabbitmq-users` discussion:

https://groups.google.com/d/topic/rabbitmq-users/LZCh9n-LuZM/discussion

To reproduce:

* Add another hostname for `127.0.0.1` in your hosts file.
* Start a broker. I used the umbrella.
* Run the following:

```
rabbitmqctl shutdown
rabbitmqctl rename_cluster_node rabbit@HOST1 rabbit@HOST2
```

You will see an error about a missing mnesia directory.

**Comment by lukebakken** [comment]
@pjk25 suggested using the "rabbitmq cli integration" test suite to test.

**Comment by lukebakken** [comment]
Fixed by #458
